common/record/_recordValidator/entityUniquenessValidator.js

```diff
@@ -9,13 +9,23 @@ const RecordValidation = require('../recordValidation')
 
 const Validator = require('../../validation/validator')
 
-const keysError = {
-  duplicateEntity: 'duplicateEntity'
+const validateEntitiesUniquenessInRecord = (survey, record) => {
```

Comment:
are you really sure you are still using  validateEntitiesUniquenessInRecord method?

Reply:
right, it's not in use anymore